### PR TITLE
feat: Add version to MonthlyAmountPerChargeResultProducedV1

### PR DESCRIPTION
**Pull request by lasrinnil**
## TODO

Please complete and remove this TODO section before entering PR ready for review state.

- [x] Title: Follow [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Write tests
- [x] Update documentation
- [ ] Update [event catalog](https://energinet.atlassian.net/wiki/spaces/D3/pages/555581556/Event+catalog) if there are relevant changes to integration events
- [ ] Inform team Outlaws if there are relevant C4 changes

### CODE REVIEW

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/WholesaleResults/WholesaleResult.cs [hunk 1]
```diff
@@ -94,4 +95,6 @@ public sealed class WholesaleResult
     public SettlementMethod? SettlementMethod { get; }
 
     public IReadOnlyCollection<WholesaleTimeSeriesPoint> TimeSeriesPoints { get; }
+
+    public long Version { get; set; }
```
Comment (dstenroejl) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/WholesaleResults/WholesaleResult.cs:99]: Is "set" necessary?

**Review by dstenroejl — COMMENTED** [review]
A single observation

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/WholesaleResults/WholesaleResult.cs (hunk 1 above)
Comment (lasrinnil) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/WholesaleResults/WholesaleResult.cs]: Nope! Removed :)

**Review by dstenroejl — APPROVED** [review]
LGTM

**Pull request merged**